### PR TITLE
Sort the component arguments in the docs

### PR DESCRIPTION
**Pull request by jonrohan**
This PR sorts the arguments found in the docs like https://primer.style/view-components/system-arguments. 

I believe this is useful because as we automate more of the arguments, they will show up sorted automatically in the docs. And sorted them will keep them consistent between pages.

https://github.com/primer/view_components/pull/612#discussion_r646073592

### CODE REVIEW

File: docs/content/components/blankslate.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ There are a few variations of how the Blankslate appears: `narrow` adds a maximu
 | `title` | `String` | `""` | Text that appears in a larger bold font. |
 | `title_tag` | `Symbol` | `:h3` | HTML tag to use for title. |
 | `icon` | `Symbol` | `""` | Octicon icon to use at top of component. |
-| `icon_size` | `Symbol` | `:medium` | One of `:small` (`16`) and `:medium` (`24`). |
+| `icon_size` | `Symbol` | `:medium` | One of `:medium` (`24`) and `:small` (`16`). |
```
Comment (joelhawksley) [inline docs/content/components/blankslate.md:132]: IMO it is nicer to have these from smallest to largest. What do you think?

Comment (jonrohan) [reply docs/content/components/blankslate.md:132]: Agreed, but that would require a lot more customized sorting comparator than here right? https://github.com/primer/view_components/pull/624/files#diff-b3a4f33411bc8a22b787fc10f0f06694f1487183b49e9add40bf17929a4c2209R7-R9

Comment (joelhawksley) [reply docs/content/components/blankslate.md:132]: Yeah, I don't know how we'd even write that. What if we added a `sort` option `one_of` and had it be false for just these cases?

File: lib/yard/docs_helper.rb [hunk 2]
```diff
@@ -3,7 +3,13 @@
 module YARD
   # Helper methods to use for yard documentation
   module DocsHelper
-    def one_of(enumerable, lower: false)
+    def one_of(enumerable, lower: false, sort: false)
```
Comment (joelhawksley) [inline lib/yard/docs_helper.rb:6]: What do you think about defaulting `sort` to `true`?

Comment (joelhawksley) [reply lib/yard/docs_helper.rb:6]: Also, maybe we should write a test for this change?

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,12 @@
 
     *Manuel Puyol*
 
+### Misc
+
+* Sort the docs arguments
```
Comment (joelhawksley) [inline CHANGELOG.md:15]:
```suggestion
* Sort documentation arguments.
```

**Review by joelhawksley — APPROVED** [review]

**Pull request merged**